CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.1.0] - 2020-11-04
+
+ * First release with support for I2C.
```

Comment:
I'd either list the whole feature set (HW / SW I2C, uC architecture abstraction, common helpers, .... Would need to think about a good wording) or just leave it of and just state: "First versioned release" or something like that.

Reply:
This good?